README.md

```diff
@@ -74,16 +76,16 @@ to emit an event, it will write to the Console and File sinks.
 
 The SSL support has been improved. Now all properties are used when creating the connection.
 
-### Breaking changes:
-- Renamed hostname to hostnames in configuration
-- Default text formatter is `CompactJsonFormatter`. The `RawFromatter` was removed from Serilog.
+### Breaking changes
 
+- Renamed `hostname` to `hostnames` in configuration.
+- Default text formatter is `CompactJsonFormatter`. The `RawFromatter` was removed from Serilog.
 
 ## Topics
 
 * [Sink Configuration Options](#sink-configuration-options)
-* [External configuration using Serilog.Settings.AppSettings](#external-configuration-using-serilogsettingsappsettings)
-* [External configuration using Serilog.Settings.Configuration](#external-configuration-using-serilogsettingsconfiguration)
+* [External configuration using Serilog.Settings.AppSettings](#external-configuration-using-serilog.settings.appsettings)
```

Comment:
The anchor links are now broken

Reply:
Reverted. Different behavior locally and on GitHub.